apps/alerting/notifications/Makefile

```
@@ -1,4 +1,4 @@
 .PHONY: generate
 generate:
-	## --crdencoding none is needed to avoid infinite loop while generating recursive models'
-	grafana-app-sdk generate -c . -g ./apis --crdencoding none
+	## --crdencoding none is needed to avoid infinite loop while generating recursive models (see routingtree.cue)
+	grafana-app-sdk generate -g ./apis --crdencoding none --nomanifest --postprocess
```

> I think the generation of a group with multiple resources could be improved: instead of the dir structure resource/$kind/$version, I would propose resource/$version/$kind. This will allow adding some common definitions, such as a struct GroupVersion, to' resource/$version`.

i think this might be what you are looking for:
```suggestion
	grafana-app-sdk generate -g ./apis --crdencoding none --nomanifest --postprocess --kindgrouping=group
```